src/test/scala/za/co/absa/hyperdrive/trigger/persistance/SensorRepositoryTest.scala

```diff
@@ -122,6 +123,25 @@ class SensorRepositoryTest extends FlatSpec with Matchers with BeforeAndAfterAll
       .map { case (sensor, _) => sensor }
   }
 
+  "sensorRepository.getChangedSensors" should "not fail on large number of sensors" in {
```

Comment:
Does this test actually fail without the fix?

Reply:
Yes

Reply:
Nice